GetStream/stream-chat-android
# [4231] Fix displaying image attachments that have failed to send V6

# V6 Counterpart of https://github.com/GetStream/stream-chat-android/pull/4234

### 🎯 Goal

closes #4231

### 🛠 Implementation details

Added a fallback to `Attachment.upload`

### 🎨 UI Changes

| Before | After |
| --- | --- |
| ![Screenshot_1664884140](https://user-images.githubusercontent.com/37080097/193820155-729573f7-712b-4431-aa5a-cd18fb437040.png) | ![Screenshot_1664884104](https://user-images.githubusercontent.com/37080097/193820264-c79be0d5-ed39-4f96-b640-9471d578a065.png) |

### 🧪 Testing

1. Open the XML app
2. Go to a channel
3. Send an image and kill the internet mid send

Result: You should see a failed message that will be correctly displayed.

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

![gif](https://media1.giphy.com/media/8vQSQ3cNXuDGo/giphy.gif?cid=ecf05e47ocjbsa35mv3xe3sx7kx42lzaszr8lj3f4hekektr&rid=giphy.gif&ct=g)
